gamedata/movedefs.lua

```diff
@@ -339,8 +339,8 @@ local moveDatas = {
 	--corprinter corvac corvacct correap corftiger armbull legsco corvoc armmerl
 	MTANK3 = {
 		crushstrength = 250,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```

Comment:
Shouldn't this be called MTANK4 now? It seemed the name was intended to match the foorptinr size,